plugins/errorlogs/frontend/public/javascripts/countly.views.js

```diff
@@ -1,83 +1,82 @@
-            $(".btn-clear-log").on("click", function() {
-                var id = $(this).data("id");
-                CountlyHelpers.confirm(jQuery.i18n.map["errorlogs.confirm-delete-" + id] || jQuery.i18n.map["errorlogs.confirm-delete"], "popStyleGreen", function(result) {
+        },
+        methods: {
+            refresh: function(force) {
+                var self = this;
+                if (force) {
+                    return $.when(countlyErrorLogs.getLogByName(this.selectLog))
+                        .then(function() {
+                            self.cachedLog = countlyErrorLogs.getLogCached();
+                        });
+                }
+            },
+            changeLog: function(value) {
+                this.downloadLink = countlyGlobal.path + "/o/errorlogs?api_key=" + countlyGlobal.member.api_key + "&app_id=" + countlyCommon.ACTIVE_APP_ID + "&download=true&log=" + value,
+                app.navigate("#/manage/errorlogs/" + value);
+                this.refresh(true);
+            },
+            clear: function() {
+                var self = this;
+                CountlyHelpers.confirm(jQuery.i18n.map["errorlogs.confirm-delete-" + self.selectLog] || jQuery.i18n.map["errorlogs.confirm-delete"], "popStyleGreen", function(result) {
                     if (!result) {
                         return true;
                     }
-                    $.when(countlyErrorLogs.del(id)).then(function(resData) {
+                    $.when(countlyErrorLogs.del(self.selectLog)).then(function(resData) {
                         if (resData.result === "Success") {
                             $.when(countlyErrorLogs.initialize()).then(function() {
-                                countlyErrorLogs.getLogByName(id, function() {
-                                    self.renderCommon();
-                                    app.localize();
+                                countlyErrorLogs.getLogByName(self.selectLog, function() {
+                                    self.refresh(true);
                                 });
                             });
                         }
                         else {
                             CountlyHelpers.alert(resData.result, "red");
                         }
                     });
-                }, [jQuery.i18n.map["common.no-dont-delete"], jQuery.i18n.map["common.yes-clear-it"]], {title: jQuery.i18n.map["errorlogs.confirm-delete-" + id + "-title"] || jQuery.i18n.map["errorlogs.confirm-delete-title"], image: "clear-api-logs"});
-            });
+                }, [jQuery.i18n.map["common.no-dont-delete"], jQuery.i18n.map["common.yes-clear-it"]], {title: jQuery.i18n.map["errorlogs.confirm-delete-" + self.selectLog + "-title"] || jQuery.i18n.map["errorlogs.confirm-delete-title"], image: "clear-api-logs"});
+            }
         }
-    },
-    refresh: function() {
-    }
-});
+    });
 
+    //register views
+    app.errorLogsView = new countlyVue.views.BackboneWrapper({
```

Comment:
i don't think we need to assign the returned value to app object.
you can simply create a variable here.
or imo the best practice is to create a function that returns the backbone view and call that function only when the route is triggered.
https://github.com/Countly/countly-server/blob/9a72eba5057bc960d078df5d32dbcdfc27de9938/plugins/vue-example/frontend/public/javascripts/countly.views.js#L737-L768

Reply:
what do you think @can-keklik

Reply:
Yeah, I've been thinking about this. It apparently works this way, too. But our practical knowledge is limited about whether reusing the BackboneWrapper causes any trouble in terms of Vue instance's lifecycle, etc. 

I think it is better to create the BackboneWrapper in a lazy fashion (and not mount to `app`) for the time being, as Prikshit suggested.